sdk/go/channels.go

```diff
@@ -235,3 +289,29 @@ func (sdk mfSDK) DeleteChannel(id, token string) error {
 
 	return nil
 }
+
+func channelsFromFile(path string, channels []Channel) error {
```

Comment:
Can you please modify this functions into `func channelsFromFile(path string) ([]Channel, error) {`